tests/journey/test_federal_senate_dataset.py

```diff
@@ -0,0 +1,34 @@
+import os
+from tempfile import gettempdir
+from unittest import TestCase, skipIf
+
+from serenata_toolbox.federal_senate.federal_senate_dataset \
+    import FederalSenateDataset
```

Comment:
[I don't care about PEP8 when it ends up making code harder to read](https://youtu.be/wf-BqAjZb8M). Let's go beyond 80 cols here?